bin/devcontainer_setup_scripts/root_setup.sh

```diff
@@ -33,28 +35,40 @@ if ! type nvim >/dev/null 2>&1; then
 	install lazygit /usr/local/bin
 	rm lazygit.tar.gz lazygit
 
-	# Installing neovim via .deb
-	# curl -LO https://github.com/neovim/neovim/releases/download/v0.9.1/nvim-linux64.deb
-	# apt install ./nvim-linux64.deb
-	# rm ./nvim-linux64.deb
-
+	NVIM_VERSION=v0.8.3
 	# .deb is not supported for 0.9.0 version upwards. More info: https://github.com/neovim/neovim/issues/22684
 	# Another options for installing neovim:
 	# https://github.com/MordechaiHadad/bob
 
-	# Installing neovim via appimage. Recommended approach: https://github.com/neovim/neovim/releases/download/v0.9.1/nvim-linux64.deb
-	# curl -LO https://github.com/neovim/neovim/releases/latest/download/nvim.appimage
-	curl -LO https://github.com/neovim/neovim/releases/download/v0.8.3/nvim.appimage
-	chmod u+x nvim.appimage
-	./nvim.appimage --appimage-extract
-	./squashfs-root/AppRun --version
-	rm nvim.appimage
+	# Install neovim in the way above only for ARM architectures
+	if [ "$(uname -m)" = "aarch64" ] || [ "$(uname -m)" = "arm" ]; then
+		# Steps defined here: https://dev.to/asyncedd/building-neovim-from-source-1794
+		apt-get update
+		apt-get install -y ninja-build gettext libtool libtool-bin autoconf automake cmake g++ pkg-config unzip curl doxygen
+		git clone https://github.com/neovim/neovim.git
+		# Versions higher than v0.8.3 present visualization problems when connected inside neovim
+		cd neovim
+		git checkout ${NVIM_VERSION}
+		make CMAKE_BUILD_TYPE=RelWithDebInfo
+		make install
+		cd -
+		rm -rf neovim
```

Comment:
This is the part of the code that installs nvim from the source (inside docker). This is slower than the installation below, which uses the nvim.appimage and is used when the architecture is not based on arm.

nvim community is asking for nvim.appimages based on arm architecture:
https://github.com/neovim/neovim/issues/15143